python-backend/main.py

```diff
@@ -107,12 +107,16 @@ async def data_socket(websocket: WebSocket):
             logger.debug(f"last convergent / limit: {limit}")
             await websocket.send_json({"limit": "Infinity" if type(limit) is Infinity else str(limit)})
 
+            # @TODO replace wide_search=True with wide_search=[1]
```

Comment:
wanted to do this but then did not receive any results back... point of clarification for them, but it's working as is now even if it may cause perf issues with more complex polys